java/bogdana_maksiuta/src/com/zoolatech/lecture3/tasks/_3/Circle.java

```diff
@@ -0,0 +1,45 @@
+package com.zoolatech.lecture3.tasks._3;
+
+import java.util.Objects;
+
+public non-sealed class Circle implements Shape {
+    final double PI = 3.1415926536;
+    int radius;
+
+    public Circle(int radius) {
+        this.radius = radius;
+    }
+
+    @Override
+    public void findPerimeter() {
+        int perimeter = (int) (radius * 2 * PI);
```

Comment:
why int and not a floating-point number?

Reply:
without reason. fixed